pack/fhv/fhvc.json

```diff
@@ -146,15 +146,296 @@
+    "illustrator": "Pixoloid Studios",
+    "name": "Vineyard",
+    "pack_code": "fhvc",
+    "position": 132,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "[free] During your turn: Engage an enemy at Vineyard. (Limit once per turn.)\n<b>Forced</b> – When an enemy enters Vineyard: Heal 1 damage from it.",
+    "traits": "Farm.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The old farm is riddled with scraps, desiccated livestock, rotten feed, and rusted farming equipment.",
+    "back_name": "Atwood Farm",
+    "clues": 1,
+    "code": "10633",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 8,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Coop",
+    "pack_code": "fhvc",
+    "position": 133,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "While there is a decoy on Coop, this location gets -2 shroud.\nEach enemy at Coop gets +1 fight and + 1 evade.\n[action]: Draw 1 set-aside Fire! treachery. Shuffle 1 copy of a set-aside Fire! treachery into the encounter deck. (Limit once per round.)",
+    "traits": "Farm.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The old farm is riddled with scraps, desiccated livestock, rotten feed, and rusted farming equipment.",
+    "back_name": "Atwood Farm",
+    "clues": 1,
+    "code": "10634",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "flavor": "The barn looks like it could collapse at any moment.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Barn",
+    "pack_code": "fhvc",
+    "position": 134,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "Enemies at Barn get +1 damage value.\nWhen Barn is revealed: Place the set-aside Ajax story asset at this location <b>([codex] Ω)</b>.",
+    "traits": "Farm.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The old farm is riddled with scraps, desiccated livestock, rotten feed, and rusted farming equipment.",
+    "back_name": "Atwood Farm",
+    "clues": 1,
+    "code": "10635",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 10,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Pasture",
+    "pack_code": "fhvc",
+    "position": 135,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "[free] During your turn, if Pasture has a trap on it: <b>Move</b> <i>(to a connecting location)</i>.\n<b>Forced</b> – After a ready, unengaged enemy moves into Pasture via a keyword, if there are no barriers at Pasture: Resolve that keyword again.",
+    "traits": "Farm.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
```

Comment:
```suggestion
    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```

Reply:
NB we've not used `<cite>` elsewhere on the DB, so please don't introduce it now - let's stick with keeping the flavor style as consistent as we can (NB2, I know that the style of Lovecraft's own name has changed across cards - H.P. Lovecraft, HP Lovecraft, H. P. Lovecraft, etc - one day I'll work through making them all the same, even if FFG can't manage that!)